client/src/assets/SearchIcon.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+export const SearchIcon = () => (
```

Comment:
Why dont you use material-ui ? I think material ui comes with icons + lots of ui components